packages/react/src/file-upload/FileUpload.stories.tsx

```diff
@@ -271,6 +271,14 @@ export const ExistingFiles: Story = {
 				name: 'police-check.pdf',
 				size: 100000,
 			},
+			{
+				name: 'police-check.pdf',
```

Comment:
Can you change this to 
```suggestion
				name: 'name: 'onclick.jpg',
```